src/api/student.js

```diff
@@ -29,6 +29,10 @@ app.post('/', (req, res) => {
                 email: s['E-mail'],
                 major: s['Major']
             };
+            if (student.name === undefined || student.name === undefined || student.status === undefined ||
```

Comment:
`if(Object.values(student).includes(undefined))` if you're into that sort of thing.